### PR TITLE
Hearthkin Batch 1 (Hearthkin, Cryo, Stone and others)

### PR DESCRIPTION
**Pull request by Kaostico**
## About The Pull Request

This is part 1 of Hearthkin DLC. I had to cut it short because it was becoming a BEHEMOTH of a PR. The next batch, which Rhaplanca will work on, will include the maps and ruins.

### What does this include?

- Hearthkin species (Of course)
- Primitive modules such as cooking, production, structures and more
- Reagent Forging
- Combat Intent
- Cryo
- S T O N E
- Xenoarcheology
- Religion module
- Sound module
- Crafting module
- Traits module
- And mooooooore

## Why It's Good For The Game

Hearthkin Must Rise To War

## Changelog

:cl: Kaostico
add: Added the hearthkin to Doppler.
add: Added primitive modules, such as cooking and structures.
add: Added Reagent Forging: the hearthkin now can avoid death easier.
add: Added cryogenics functions to the game. Rooms TBA for station.
add: Added xenoarcheology. For gacha's brain scratching dopamine.
add: Added combat intent.
/:cl:

### CODE REVIEW

**Comment by Kaostico** [comment]
Linters is throwing an indentation error. I'm unable to fix it because git is not noticing the difference between spaces and tab indentation.
I would really appreciate some help! =D

**Comment by Kaostico** [comment]
I just noticed there's some things that I didn't account for. I'll look at this tomorrow with a fresh set of eyes.

**Comment by Kaostico** [comment]
I was somehow able to finally finish this PR! It's ready for review!

File: modular_doppler/reagent_forging/code/forge_weapons.dm [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+
+	var/obj/item/ammo_casing/arrow/converted_arrow = new spawned_item(get_turf(src))
+	transfer_fingerprints_to(converted_arrow)
+	remove_item_from_storage(user)
+	user.put_in_hands(converted_arrow)
+	qdel(src)
+
+#define INCREASE_BLOCK_CHANCE 2
+
+/obj/item/forging/reagent_weapon/bokken
```
Comment (honkpocket) [inline modular_doppler/reagent_forging/code/forge_weapons.dm:297]: you're as beautiful as the day i lost you

**Review by CliffracerX — APPROVED** [review]
SPEEDMERGE THIS *NOW*

**Pull request merged**